### ISSUE TITLE
Performance plotting functions

### ISSUE THREAD

**Issue by dschoenig** [maxkroeschel/activitytools#10]
Making plot functions work with `activity` objects (c4fd72f5966f320f3e78867f5796098a857fab85) decreases performance by several orders of magnitude if the `activity` object is large.

**Comment by dschoenig** [comment]
Problem is connected to passing data to manipulate. 14c86f0 solves it for plotting states and thresholds but not activity

**Comment by dschoenig** [comment]
Minimal example:

```

plot_activity_2 <- function(activity,
                thresholds,
                gps,
                ...){
  plot_activity(activity=activity,
                thresholds=thresholds,
                gps=gps,
                ...)
}

# Package function - fast
plot_activity(activity = deer$activity_data, 
              thresholds = deer$activity_thresholds_aggregated, 
              gps = deer$gps_data, 
              animal_id = 1)

# Slow
plot_activity_2(activity = deer$activity_data, 
                thresholds = deer$activity_thresholds_aggregated, 
                gps = deer$gps_data, 
                animal_id = 1)
```

**Comment by dschoenig** [comment]
Fixed in 629c5cf with workaround to prevent `manipulate` from slowing down.
